openqa-trigger-bisect-jobs

```diff
@@ -298,11 +305,20 @@ def main(args):
                 "MAINT_TEST_REPO=",
             ]
         )
+        import pdb
+        #pdb.set_trace()
+        try:
+            out = openqa_clone(
+                params,
+                args.dry_run,
+            )
+        except RuntimeError as err:
+            err_comment = str(err)
+            log.error(err_comment)
+            #openqa_comment(base_url, job_id, err_comment, args.dry_run)
+            #print(f"get here")
+            sys.exit()
```

Comment:
If the `openqa_clone` fails for a different reason it should still exit with the original return code. We only want to handle the mentioned case.

Reply:
`mentioned case`?
So you want to pass the `returncode`?

Reply:
`openqa-clone-job` can fail for several reasons. Only if the error says "Current job ... will fail, because the repositories for the below updates are unavailable" we want to treat it as non-fatal.
In all other cases it should exit with something other than zero.

Reply:
that will take some time to figure it out. I see openqa_clone_job dies. I will take a look

Reply:
Maybe you can just re-raise the original exception in those other cases.

Reply:
I submitted my approach but it is untested and I am not sure if it works actually in a real scenario. I assume that the error will be raised by `SubprocessError`